dataformat-all/pom.xml

```diff
@@ -47,6 +47,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
+        <version>3.4.0</version>
```

Comment:
Any version lower than 3.4.0 produces the following maven build error:

`Error creating shaded jar: Problem shading JAR ~/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.15.2/jackson-core-2.15.2.jar entry META-INF/versions/19/com/fasterxml/jackson/core/io/doubleparser/FastDoubleSwar.class: java.lang.IllegalArgumentException: Unsupported class file major version 63`

Reply:
❓ Can we use the latest version here?